test_scripts/Polices/build_options/032_ATF_P_TC_PTU_Retry_Sequence_Retry_Timeout_Expiration_HTTP.lua

```diff
@@ -103,14 +100,15 @@ function Test:TestStep_Retry_Timeout_Expiration()
   time_wait[5] = sec_btw_ret[3] + sec_btw_ret[4] + timeout_after_x_seconds
   time_wait[6] = sec_btw_ret[4] + sec_btw_ret[5] + timeout_after_x_seconds
   total_time = (time_wait[1] + time_wait[2] + time_wait[3] + time_wait[4] + time_wait[5] + time_wait[6])*1000
+  print("Wait " .. total_time .. "msec")
```

Comment:
@istoimenova, In lines 97 - 101 time_wait[x] calculated incorrectly
According to req-t formula is the following:
```
Example:

        "timeout_after_x_seconds":30,
        "seconds_between_retries":[1, 5, 25, 125, 625]

SDL->app ID: send SnapshotPT via OnSystemRequest
wait during timeout (e.g. 30s)
PTU not received
SDL->app ID: send SnapshotPT via OnSystemRequest
wait during t1 + timeout (e.g. 1s + 30s=31s)
PTU not received
SDL->app ID: send SnapshotPT via OnSystemRequest
wait during t2 + t1 + timeout (e.g. 5s + 31s + 30s=66s)
PTU not received
SDL->app ID: send SnapshotPT via OnSystemRequest
wait during t3 + t2 + timeout (e.g. 25s + 66s + 30s=121s)
etc.
```

Please use the correct one from 032_ATF_P_TC_PTU_Restarting_Policy_Table_Exchange_HTTP.lua script